Cargo.toml

```diff
@@ -62,7 +62,7 @@ serde = "1.0.0"
 serde_json = "1.0.0"
 
 # Akamai example
-tokio-core = "0.1"
+tokio = "0.1"
```

Comment:
Would you mind just setting this to `0.1.8`? I'm trying to move towards minimal versions compiling. I'm not sure what the actual minimal version here that works is, but just going with `0.1.8` is fine.

Reply:
Done! 60d4d02bddd4b0db1b087e856d38224968482f75